description/urdf/nanoscan3/nanoscan3_macro.xacro

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+
+  <xacro:include filename="$(find sick_safetyscanners2)/description/urdf/common.xacro" />
+
+  <!-- joint offsets when mounted on frame -->
+  <xacro:property name="base_x" value="0.0803"/>
+  <xacro:property name="base_y" value="-0.053"/>
+
+  <xacro:property name="sensor_y" value="${-base_y}"/>
+  <xacro:property name="sensor_z" value="0.0315"/>
+  <xacro:property name="sensor_rot_z" value="${radians(-90)}"/>
+
+  <xacro:macro name="nanoscan3" params="prefix name parent *origin mounting_kit">
```

Comment:
If we can set the name, then we don't need to prefix. This is then duplicated and reduces understanding.

```suggestion
  <xacro:macro name="nanoscan3" params="name parent *origin mounting_kit">
```